src/content/extensions/multipleroutes/README.md

```diff
@@ -1,3 +1,12 @@
-This extension, once installed, will disable the option for multiple network routes in WebRTC. This will prevent any website from accessing the local machine's private IP addresses or public ones which should have been hidden by the usage of VPN. The only address left for WebRTC will be the same one used for http traffic.
+## Chrome WebRTC Network Limiter
+Disables use of multiple network routes in Chrome’s WebRTC implementation.
 
-Note that the installation of this extension could have negative impact on WebRTC real time media as the route might not be the optimized one. Worst case, if no STUN/TURN server is specified, WebRTC will fail completely as there is no local address for remote peer to connect to.
+This configures WebRTC to not use certain IP addresses:
+- IP addresses not visible to the public internet (e.g. addresses like 192.168.1.2)
```

Comment:
Thanks for catching this, I forgot to update this copy of readme.md.

We don't check the range of private IP for ipv4 at least so not sure we can
make that statement. It's really host candidate.



Thanks,
Guowei

On Thu, Jul 23, 2015 at 8:07 AM, Christoffer Jansson <
notifications@github.com> wrote:

> In src/content/extensions/multipleroutes/README.md
> <https://github.com/webrtc/samples/pull/584#discussion_r35331537>:
>
> >
> > -Note that the installation of this extension could have negative impact on WebRTC real time media as the route might not be the optimized one. Worst case, if no STUN/TURN server is specified, WebRTC will fail completely as there is no local address for remote peer to connect to.
> > +This configures WebRTC to not use certain IP addresses:
> > +- IP addresses not visible to the public internet (e.g. addresses like 192.168.1.2)
>
> Since the concept of public and private addresses is used throughout,
> maybe add "private" like this (e.g. private addresses like 192.168.1.2) ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/samples/pull/584/files#r35331537>.
>